docs/dev/table/hive/read_write_hive.md

```diff
@@ -141,11 +141,11 @@ Flink SQL> INSERT OVERWRITE myparttable PARTITION (my_type='type_1') SELECT 'Tom
 
 We have tested on the following of table storage formats: text, csv, SequenceFile, ORC, and Parquet.
 
-# ------ ORC Vectorized Optimization ------ 
+### ORC Vectorized Optimization
 Optimization is used automatically when the following conditions are met:
 
 - Columns without complex data type, like hive types: List, Map, Struct, Union.
-- Hive version greater than or equal to version 2.0.0.
+- Hive version greater than or equal to version 2.3.0.
```

Comment:
Shall we also mention using this feature requires `flink-orc` jar in classpath?

Reply:
In my opinion, user don't need do anything, just add `flink-connector-hive` to his pom, and will have orc in the jar-with-deps.

Reply:
maybe we can pack this classes into `flink-connector-hive`.

Reply:
I think that's also viable. And if we go that way, we can pack `flink-hadoop-compatibility` as well.

Reply:
+1 to pack the two jars